Automattic/jetpack
# ConnectionBanner: allow "icon" prop to be an element

Fixes #21706

#### Changes proposed in this Pull Request:

- Allow `<ConnectionBanner>` prop `icon` to be an element type, instead of just a Gridicon string name.
- Update `<UserUnlinked>` notice to use "W" mark element instead of a Gridicon element. Fixes #21706

#### Jetpack product discussion

#21706

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:

- Connect a site with a user connection
- Create a secondary admin
- Login as the secondary admin
- Observe the updated logo in the notice on the Jetpack dashboard:

**Before:**

![before-desktop](https://user-images.githubusercontent.com/15803018/144672149-c00a1a34-1080-4373-8b94-be5180fbec8d.png)
![before-mobile](https://user-images.githubusercontent.com/15803018/144672154-d45759e4-44a9-4ef0-9db1-8415c0c8f8ea.png)

**After:**

![Screen Shot 2021-12-09 at 5 15 31 PM](https://user-images.githubusercontent.com/15803018/145495741-757e598f-1986-4dba-9aff-d59e6fabf7f1.png)
